src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -689,6 +696,8 @@ void OrbitMainWindow::UpdateProcessConnectionStateDependentWidgets() {
 void OrbitMainWindow::ClearCaptureFilters() { filter_panel_action_->ClearEdits(); }
 
 void OrbitMainWindow::UpdateActiveTabsAfterSelection(bool selection_has_samples) {
+  if (absl::GetFlag(FLAGS_time_range_selection)) return;
```

Comment:
Are you going to get rid of this function soon? If not, at least consider adding a comment

Reply:
yes it will be part of the clean up